code/game/turfs/turf.dm

```diff
@@ -8,6 +8,10 @@
 	var/slowdown = 0 //negative for faster, positive for slower
 	var/transparent_floor = FALSE //used to check if pipes should be visible under the turf or not
 
+	// set if the turf should appear on a different layer while in-game and map editing, otherwise use normal layer
+	var/real_layer = TURF_LAYER
```

Comment:
```suggestion
	/// set if the turf should appear on a different layer while in-game and map editing, otherwise use normal layer
	var/real_layer = TURF_LAYER
```
Could be an autodoc.